### DISCUSSION TITLE
Allow receiver address to be the one simple login redirects to by toggleable button.

### DISCUSSION THREAD

**Discussion by Chrattac** [simple-login/app#1240] [Feature Requests]
Hey,

I have been using my mail with several different addresses directing to mailbox. By standard I have automatically sorted the mail the way mails coming to address X goes to folder X and to Y to folder Y. Y is mailbox for important mail that I read instantly and get notifications whenever I get mail into it. The X's purpose is for mails with less of an importance ie. newsletters, order confirmations etc. Basically that one is planned to serve purpose of what I have been using my gmail account for ie. mailbox that I give whenever mail is required but I don't care anything coming to it besides some order confirmations and "verify your mail" -messages (and have rule to delete mail after automatically instead of having +100'000 unread mails in inbox like I do in gmail). 

I am in process of ditching gmail (and google) completely due to google's well known bad data practises related to it even if there is nothing of importance coming into it. 

The problem is that for the best results I would love to use simplelogin but the fact whatever alias I use that's always in receiver field along with inverse alias of sender. Could there be toggleable button to make the mail coming in to be processed such that the receiver is the one it is redirected to instead of the original alias (ie asdfafa.simplelogin@simplelogin.co redirects to xxx@yyy.com will show that one as a receiver instead of asdfafa.simplelogin@simplelogin.co)? This would help a lot since I would use simple login for both more important mail as well as for the less important one and adding new rule for each alias is such a hassle. Especially since in my opinion it is better to use more aliases than less.

**Comment by nguyenkims** [comment]
@Chrattac If we use the mailbox address as To: field, then it's hard to distinguish emails that are sent to different aliases that belong to this mailbox.

Currently SimpleLogin sets several additional headers like `X-SimpleLogin-Envelope-From`, `X-SimpleLogin-Envelope-To`, etc and we can also add a header `X-SimpleLogin-Mailbox-To`, maybe that would work for your filter?

**Comment by Chrattac** [comment]
Hey,

There may be some way to filter coming mail with the form if the forwarding address can be shown somehow. I need to investigate the UI and see if I can do something with filters right now. The problem is just that then I have to set up every alias separately instead of just creating them with the extension on the go.

Is the "hard to distinguish" on Simple Login's side? If it is just for original receiver's side issue, the toggle would be sufficient with notion that it would break reply as I personally would use this mainly to filter mails I most likely don't care to even read and will autodelete anyway (Ie. registering to various web services, order confirmations etc.. which most of them are no-reply) to make them go to "no notification, mark as read, delete" -mailbox in case those addresses start seeing spam as some services think it is OK to auto add customers or registrees into mailing lists. Still some of these are services I can't just turn alias off due to occasional mail I need to read (eg. order confirmation, password reset).

**Reply by nguyenkims** [reply]
The "hard to distinguish" is for SimpleLogin users indeed, we can't know which alias an email is sent to if the email has `To` set to mailbox address.

**Reply by devtoor** [reply]
I think the `From` or the `reverse alias` can be used to determine which alias. The login alias and the reverse alias is a 1:1 representations (basic translation). Added headers should be minimize for security and privacy.
